### DISCUSSION TITLE
How to assign Multi-Bucket-Access to one Noobaa Account after v5.12.4

### DISCUSSION THREAD

**Discussion by schwienbier** [noobaa/noobaa-operator#1168] [Q&A]
Dear Noobaa Developers,

according to this merged [PR](https://github.com/noobaa/noobaa-operator/commit/e03f0cae7aefdb4fa4f05e3fe0281c17148fc780), "allowed_buckets" are removed from noobaa-account-crd. Do you know if there is any alternative way to assign Multi-Bucket-Access to one noobaa account after v5.12.4? Thank you. 

Best regards

**Comment by nimrod-becker** [comment]
Using the S3 way which is S3 Bucket Policies

**Reply by schwienbier** [reply]
Hi @nimrod-becker , we can not assign AWS bucket policies to a Noobaa account, right? Can we define Noobaa native bucket policies, and then assign those bucket policies to a Noobaa account? Thank you.

**Reply by nimrod-becker** [reply]
The Bucket Policies are applied to the buckets, and are referencing an account.
If the buckers in use are rather static (i.e. not being created/deleted all the time) its a one time to set the appropriate policies.
If they are constantly being created/deleted by the application(s) then a policy should be set after each creation
